src/main/java/seedu/address/model/person/Age.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's age in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidAge(String)}
+ */
+public class Age {
```

Comment:
Since age is constantly changing, it might be good to input birthdates instead. Then we can get the age through another method such as taking LocalDateTime.now()  - birthdate. This would reduce the need for resubmitting the age.

Reply:
Agreed, definitely a better idea!! Thanks for the suggestion, will make the change now :)